lua/weapons/hands.lua

```diff
@@ -0,0 +1,78 @@
+
+AddCSLuaFile()
+
+local SWEP = { Primary = {}, Secondary = {} }
+local baseClass = baseclass.Get("weapon_base")
+
+SWEP.Author     	       = "PAC3 Team & TAS Team"
+SWEP.Contact      	       = "https://taservers.com/"
+SWEP.Purpose      	       = ""
+SWEP.Instructions          = "Right-click while walking and crouching to toggle the crosshair"
+SWEP.PrintName             = "Hands"
+SWEP.IconOverride          = "entities/weapon_fists.png"
+SWEP.Spawnable    	       = true
+SWEP.AdminSpawnable	       = false
+
+SWEP.AutoSwitchTo	       = true
+SWEP.AutoSwitchFrom	       = true
+SWEP.Weight 		       = 1
+SWEP.DrawAmmo              = false
+SWEP.DrawCrosshair	       = true
+SWEP.DrawWeaponInfoBox     = true
+SWEP.SlotPos      	       = 1
+SWEP.Slot         	       = 1
+SWEP.HoldType              = "normal"
+SWEP.ViewModel             = "models/weapons/c_arms.mdl"
+
+SWEP.Primary.ClipSize      = -1
+SWEP.Primary.DefaultClip   = -1
+SWEP.Primary.Automatic     = false
+SWEP.Primary.Ammo          = "none"
+
+SWEP.Secondary.ClipSize    = -1
+SWEP.Secondary.DefaultClip = -1
+SWEP.Secondary.Automatic   = false
+SWEP.Secondary.Ammo        = "none"
+
+function SWEP:DrawHUD() end
+function SWEP:DrawWorldModel() end
+function SWEP:DrawWorldModelTranslucent() end
+function SWEP:CanPrimaryAttack() return false end
+function SWEP:CanSecondaryAttack() return false end
+function SWEP:Reload() return false end
+function SWEP:Holster() return true  end
+function SWEP:ShouldDropOnDie() return false end
+
+local weaponSelectionColor = Color(255, 220, 0, 255)
+function SWEP:DrawWeaponSelection(x, y, w, t, a)
+	weaponSelectionColor.a = a
+	draw.SimpleText("C", "creditslogo", x + w / 2, y, weaponSelectionColor, TEXT_ALIGN_CENTER)
+	baseClass.PrintWeaponInfo(self, x + w + 20, y + t * 0.95, alpha)
+end
+
+function SWEP:Initialize()
+	if self.SetHoldType then
+		self:SetHoldType("normal")
+	else
+		self:SetWeaponHoldType("normal")
+	end
```

Comment:
This isn't officially deprecated on the wiki, but does mention to never use it:
![image](https://github.com/TAServers/gmod-utils/assets/49565664/ae109f44-764c-4cfc-b1ca-a335d72743b3)

I think we can remove the condition and just have `self:SetHoldType`